src/context/auth.js

```diff
@@ -59,6 +64,7 @@ const AuthProvider = ({ children }) => {
 
 	const value = {
 		token,
+    userId,
```

Comment:
Not sure if there is some weird formatting going on here in github, or if this needs indenting in the code.

Reply:
Looks like some lines had used tabs instead of spaces, will fix this with next commit